packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -308,7 +311,8 @@ export default {
             tenantConfigs,
             hasCheckoutLoadedSuccessfully: true,
             shouldShowSpinner: false,
-            isLoading: false
+            isLoading: false,
+            hasAccessForbiddenError: false
```

Comment:
I might be wrong, but I think I preferred the approach you showed me when we were pairing of using the custom `AccessForbiddenError`. 
The problem I see with the current approach is that we can have mutually exclusive scenarios (e.g. `hasCheckoutLoadedSuccessfully && hasAccessForbiddenError`, and at the same time, if we want to handle another type of error in the future, this will grow significantly in complexity.

Reply:
Sure, I've updated the code :white_check_mark: